code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/process/InstancePurgePostListener.java

```diff
@@ -0,0 +1,63 @@
+package io.cattle.platform.servicediscovery.process;
```

Comment:
We used to have the logic and removed ~7 weeks ago
https://github.com/rancher/cattle/pull/1513